infra/src/main/resources/exception/exceptionInfo.json

```diff
@@ -196,6 +196,35 @@
     "id": 34,
     "reason": "The selected Region is not valid.",
     "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ProvisioningFailureException",
+    "id": 35,
+    "reason": "An error occurred handling your request. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.UnspecifiedProvisioningFailureException",
+    "id": 36,
+    "reason": "An unspecified error occurred handling your request. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
+  },
+  {
+    "exception": "com.redhat.service.smartevents.infra.exceptions.definitions.user.ProvisioningTimeOutException",
+    "id": 37,
+    "reason": "Your request was not complete within the configured time-out. Please contact the Help Desk for support quoting the date and time of this error.",
+    "type": "USER"
```

Comment:
@manstis I think we need some clarity on the usage of `type` here. `type` was (I believe) intended to be used as a marker for an error directly related to something the user has done, and returned synchronously from the call the API of SmartEvents.  Good examples being: You're missing a field, you have a bad template definition.

On that understanding of `type`, it doesn't feel appropriate to use it here as these are asynchronous errors and most of them may not be related to what the user has done. For example: Being unable to provision a `ReplicaSet` may be zero fault of the user, and I'm not totally convinced we want to expose that much data to them anyway.

I would personally say that at this point, all errors from the Shard are `platform` errors and the role of the Fleet Manager is to map the specific error from the Data Plane to a useable `status_message`

Reply:
I had originally had them all extend `InternalPlatformException` as I agree; it makes more sense.

When error codes are expanded to messages I lookup the `reason` from the applicable `BridgeError`. `ErrorService` only supports looking up `USER` errors hence my change (to extend `ExternalUserException`). I can change back to `InternalPlatformException` and add a new method.